src/LendingPool.sol

```diff
@@ -183,16 +182,14 @@ contract LendingPool is Owned, TrustedProtocol, DebtToken {
      * @param receiver The address of the receiver of the underlying ERC-20 tokens
      */
     function withdrawFromLendingPool(uint256 assets, address receiver) public {
-        _syncInterests();
-
         require(realisedLiquidityOf[msg.sender] >= assets, "LP_WFLP: Amount exceeds balance");
```

Comment:
isn't this checked by line 187's safemath? if so, see if it's cheaper to remove the require or put line 187 as unchecked @Thomas-Smets

Reply:
Moved to separate issue